dss/stepfunctions/gscopyclient/implementation.py

```diff
@@ -84,6 +84,8 @@ def run(self) -> dict:
                 response = dst_blob.rewrite(src_blob, token=state.get(_Key.TOKEN, None))
                 if response[0] is None:
                     state[Key.FINISHED] = True
+                    async_key = f"{Replica.gcp.name}/{self.destination_key}"
```

Comment:
I noticed an inconsistency between the s3copyclient and gscopyclient in that the s3copyclient has a function to construct the `_async_key` that provides a default value, but the gscopyclient does not. This may (or may not) be correct and appropriate, I did not dig in deeply to determine that. Here I am just noting the apparent difference/inconsistency.